.github/workflows/openapi-publish.yml

```diff
@@ -0,0 +1,27 @@
+name: Publish OpenAPI specification to documentation
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - 'swagger.json'
+jobs:
+  publish-to-docs:
+    needs: make-openapi-spec
```

Comment:
Wouldn't it be easier to just use actions/checkout@v2 here, to use the `swagger.json` from the repo instead of downloading an artifact? 
If I understand it right, there now will be a `swagger.json` in the repo, right?